### PR TITLE
Added RabbitMQ Service to compose file (docker-compose.yml

### PR DESCRIPTION
**Pull request by ke45xumo**
Added RabbitMQ Container to docker-compose.yml

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -197,3 +197,21 @@ services:
...
+# -------------------- RABBIT-MQ -------------------------------------------
+  rabbitmq:
+    image:  rabbitmq:latest
+    environment: 
+      RABBITMQ_ERLANG_COOKIE: 'S0m3_R4bBi7_C0ok13'
+      RABBITMQ_DEFAULT_USER: 'rabbit_adm'
+      RABBITMQ_DEFAULT_PASS: 'R4bb!7_4DM_p4SS'
+    labels:
+        - "traefik.enable=true"
+        - "traefik.http.routers.to-core.rule=PathPrefix(`/api/rabbitmq`)"
```
Comment (georg-schwarz) [inline docker-compose.yml]: Can we name it /api/amqp ?

**Comment by georg-schwarz** [comment]
@ke45xumo please make sure to sign the CLA, otherwise we cannot accept your contribution, unfortunately ..

**Comment by georg-schwarz** [comment]
Please remove the code from the other PR again. This PR should be about adding the RabbitMQ to the system, not separating transformations and notifications!

File: docker-compose.yml (hunk 1 above)
Comment (ke45xumo) [reply docker-compose.yml]: solved with new commit

**Pull request closed without merging**